src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UITests/Presentation/SampleChooserViewModel.cs

```diff
@@ -1044,20 +1060,17 @@ private async Task SetFile<T>(string key, T value)
 
 		private async Task<T> GetFile<T>(string key, Func<T> defaultValue = null)
 		{
-#if !UNO_REFERENCE_API
+#if !__NETSTD_REFERENCE__
 			string json = null;
 
 			using (await _fileLock.LockAsync(CancellationToken.None))
 			{
 				try
 				{
-					var folderPath = ApplicationData.Current.LocalFolder.Path;
-					var filePath = Path.Combine(folderPath, SampleChooserFileAddress + key);
-					if (File.Exists(filePath))
-					{
-						json = File.ReadAllText(filePath);
-					}
-					
+					var folder = ApplicationData.Current.LocalFolder;
+					// GetFileAsync ensures the filesystem has been loaded on WASM.
+					var file = await folder.GetFileAsync(SampleChooserFileAddress + key);
+					json = File.ReadAllText(file.Path);
```

Comment:
Could you use `FileIO.ReadTextAsync` instead of the two lines?

Reply:
Done!

I wasn't aware of that API, thanks for letting me know!